token-upgrade/cli/Cargo.toml

```diff
@@ -20,6 +20,7 @@ solana-logger = "2.0.3"
 solana-remote-wallet = "2.0.3"
 solana-sdk = "2.0.3"
 spl-associated-token-account = { version = "4.0.0", path = "../../associated-token-account/program", features = ["no-entrypoint"] }
+spl-associated-token-account-client = { version = "1.0.0", path = "../../associated-token-account/client" }
```

Comment:
Here you added the client but did not remove the program import. I think we can lose the program!

Reply:
ah right, thanks!